### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by ozgunozerk**
Please check the below comment, I'm not sure of it

### CODE REVIEW

**Comment by nazar-pc** [comment]
We need to include https://github.com/subspace/subspace/pull/581 or else it will not work.
Also take a look at https://github.com/subspace/subspace/pull/574 and replicate node key changes. Basically when we know that node has stuck at a particular block height we need to reset network keys because there is a high probability that node is banned by the rest of the network and will not be able to reach to anyone to sync.

File: src-tauri/src/node.rs [hunk 1]
```diff
@@ -119,8 +120,35 @@ async fn create_full_client<CS: ChainSpec + 'static>(
...
 
-    subspace_service::new_full::<subspace_runtime::RuntimeApi, ExecutorDispatch>(config, true)
-        .map_err(Into::into)
+    let config_dir = config
+        .base_path
+        .as_ref()
+        .map(|base_path| base_path.config_dir("subspace_gemini_1b"));
+
+    // subspace_service::new_full::<subspace_runtime::RuntimeApi, ExecutorDispatch>(config, true)
+    //     .map_err(Into::into)
```
Comment (nazar-pc) [inline src-tauri/src/node.rs:129]: I think this can be removed

**Review by nazar-pc — APPROVED** [review]
Looks fine to me, were there any issues with it so it wasn't released yesterday?

**Comment by ozgunozerk** [comment]
> Looks fine to me, were there any issues with it so it wasn't released yesterday?

I think because it was not approved till today

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**